docs/benchmarks.md

```diff
@@ -5,57 +5,66 @@
 ## Test setup:
 
 * Infra: GCP VM
-* VM Type: n2-standard-48
-* VM Bandwidth: 32Gbps
-* OS: debian-11
-* VM location: us-west1-c
+* VM Type: n2-standard-96
+* OS:  ubuntu-20.04
+* VM Bandwidth: 100Gbps
+* VM location: us-west1-b
+* Disk Type: SSD persistent disk
 * GCS Bucket location: us-west1
 * Framework: FIO
 
 ### FIO Spec
-* Test runtime: 60sec
-* Thread count: 40
-* Block Size: 256KB for 256KB files and 1MB for all other files.
-
-### GCSFuse command
+* FIO creates a number of threads or processes doing a particular type of I/O action specified by the user.
+* The use of fio is to write a job file matching the I/O load one wants to simulate.
+* For testing different sizes of the file, change file size parameters.
+* BlockSize defines how large size we are issuing for I/O. For 256kb we used 16k block size other than that we used 1M blocksize.
+* We have a fsync parameter for writes that defines fio will sync the file after every fsync number of writes issued. When the Writefile operation called GCSFuse will write the data to disk. When the SyncFile operation called GCSFuse will write the data from disk to GCS bucket. So after fsync write operation GCSFuse will write the data into the GCS bucket.
 ```
-gcsfuse --implicit-dirs --stat-cache-ttl=60s --type-cache-ttl=60s 
---client-protocol=http1 --max-conns-per-host=100 <bucket-name> <path-to-mount-point>
+gcsfuse --implicit-dirs  --client-protocol=http1 --max-conns-per-host=100 <bucket-name> <path-to-mount-point>
 ```
-## Reads
-### Sequential reads
 
-| File Size | Bandwidth in GiB/sec | IOPS | Avg Latency (msec) |
-|-----------|----------------------|------|--------------------|
-| 256KB     | 0.47                 | 1875 | 1350               |
-| 1MB       | 1.56                 | 1554 | 1623               |
-| 2MB       | 2.27                 | 2279 | 1110               |
-| 5MB       | 3.75                 | 3796 | 670                |
-| 10MB      | 4.02                 | 4071 | 625                |
-| 50MB      | 4.37                 | 4433 | 574                |
-| 100MB     | 4.33                 | 4390 | 579.94             |
-| 200MB     | 4.31                 | 4371 | 582.89             |
-| 1GB       | 4.32                 | 4383 | 580.67             |
+### Sequential Write
```

Comment:
please use a header first write and then subheader sequential writes and random writes. Same for reads.